src/ado2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -83,12 +83,12 @@ public async Task Handle(MigrateRepoCommandArgs args)
         if (RepositoryMigrationStatus.IsFailed(migrationState))
         {
             _log.LogError($"Migration Failed. Migration ID: {migrationId}");
-            _log.LogInformation($"Migration log available at {migrationLogUrl} or by running `gh {CliContext.RootCommand} download-logs --github-target-org {args.GithubOrg} --target-repo {args.GithubRepo}`");
+            _log.LogInformation($"Migration log available at {migrationLogUrl} or by running `gh ado2gh download-logs --github-target-org {args.GithubOrg} --target-repo {args.GithubRepo}`");
```

Comment:
Perhaps we could continue to use `CliContext.RootCommand` like this?

```suggestion
            _log.LogInformation($"Migration log available at {migrationLogUrl} or by running `{CliContext.RootCommand} download-logs --github-target-org {args.GithubOrg} --target-repo {args.GithubRepo}`");
```

If you dig, let's do this for bbs2gh and gei :+1:

Reply:
I didn't mean to change this 😓 I meant to just update tests - serves me right for working on this just before going to bed!

Reply:
Reverted in 92eab21